arangod/Pregel/Conductor/LoadingState.cpp

```diff
@@ -0,0 +1,58 @@
+#include "LoadingState.h"
+
+#include "Pregel/Algorithm.h"
+#include "Pregel/Conductor.h"
+#include "Metrics/Gauge.h"
+#include "Pregel/Conductor/State.h"
+#include "Pregel/PregelFeature.h"
+#include "Pregel/WorkerConductorMessages.h"
+
+using namespace arangodb::pregel::conductor;
+
+#define LOG_PREGEL(logId, level)          \
+  LOG_TOPIC(logId, level, Logger::PREGEL) \
+      << "[job " << conductor._executionNumber << "] "
+
+Loading::Loading(Conductor& conductor) : conductor{conductor} {
+  conductor.updateState(ExecutionState::LOADING);
```

Comment:
Is this going to be removed again later? This looks like we have 2 different states now.

Reply:
Yes, definitely, the ExecutionState will be removed when the refactoring is done. Currently we still need it at some places.